model/build.gradle

```diff
@@ -5,7 +5,7 @@ compileTestJava.options.compilerArgs.add '-parameters'
 
 dependencies {
     compile("com.google.gwt.inject:gin:$ginVersion")
-
+    compile "com.fasterxml.jackson.core:jackson-annotations:$jacksonVersion"
```

Comment:
Why is this needed....existing `jackson-databind` has a dependency on this and model module compiles fine....please undo